network/802.11/hccap.ksy

```diff
@@ -0,0 +1,57 @@
+meta:
+  id: hccap
+  title: Hashcat capture old format
+  license: Unlicense
+  file-extension: hccap
+  application:
+    - Hashcat
+    - aircrack-ng
+  endian: le
+  encoding: utf-8
+  imports:
+    - /network/mac_address
+    - /network/802.11/eapol_key
+    - /network/802.11/wpa2_nonce
+doc: |
+  Native format of Hashcat password "recovery" utility
+  A sample of file for testing can be downloaded from https://web.archive.org/web/20150220013635if_/http://hashcat.net:80/misc/example_hashes/hashcat.hccap
+doc-ref: https://hashcat.net/wiki/doku.php?id=hccap
+seq:
+  - id: records
+    type: hccap
+    repeat: eos
+types:
+  hccap:
+    seq:
+      - id: essid
+        type: strz
+        size: 36
+        
+      - id: ap_mac
+        type: mac_address
+        doc: the bssid(MAC) of the access point 
+      
+      - id: stantion_mac 
+        type: mac_address
+        doc: the MAC address of a client connecting to the access point 
+        
+      - id: stantion_nonce
+        type: wpa2_nonce
+      
+      - id: ap_nonce
+        type: wpa2_nonce
+        
+      - id: eapol
+        type: eapol_key
+        
+      - id: eapol_size
+        type: u4
+        doc: size of eapol
```

Comment:
If it's size of something called "eapol", it should be `len_eapol`.

Reply:
It is not used for parsing the file format.

Reply:
I think it should be — as far as I understand, it is the size of meaningful portion of 256-byte array. Everything else there would be just random garbage, that you don't really want to use as a "key".